m4_sessions/1705-inning/group_five.md

```diff
@@ -16,3 +14,45 @@ To make students realize the psychological and sociological impacts of social me
 * Jon W.
 * Mickey Fen
 * Spencer Carter
+
+## SESSION OUTCOMES
+
+* Realization of implications of social media on yourself and others
+
+* Making people think about their social presence (+/-)
+
+* Is Social Media a direct representation of who you are
```

Comment:
Great question to consider--might be one to come back to at the end.  Is social media a representation of who you are as a developer?  As a person?